### ISSUE TITLE
Focus state on textfield and date-picker is broken

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1254]
### Description of the bug

Seems like focus on our textfields are broken after the last update from the design system. Without the dark border we are breaking [2.4.7 Focus visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)

Should follow the same style as the other components

<img width="963" alt="Skjermbilde 2023-06-16 kl  09 41 32" src="https://github.com/Altinn/app-frontend-react/assets/33222679/5c1c2204-f3b2-454d-8c2f-959acf1da501">

### Steps To Reproduce

Go to any app with text field and click or tab to a field and see that the dark border is missing

**Comment by bjosttveit** [comment]
@paal2707 is this solved in the new datepicker? It is at least not an issue in text fields anymore

**Comment by paal2707** [comment]
> [@paal2707](https://github.com/paal2707) is this solved in the new datepicker? It is at least not an issue in text fields anymore

Yes this is fixed.
